sycl/doc/extensions/experimental/sycl_ext_oneapi_complex.asciidoc

```diff
@@ -0,0 +1,334 @@
+
+            /// Divides complex numbers z and w and returns the value.
+            friend complex<value_type> operator/(const complex<value_type> &z, const complex<value_type> &w);
+            /// Divides complex number z and real y and returns the value.
+            friend complex<value_type> operator/(const complex<value_type> &z, value_type y);
+            /// Divides real x and complex number w and returns the value.
+            friend complex<value_type> operator/(value_type x, const complex<value_type> &w);
+
+            /// Compares complex numbers z and w and returns true if they are the same, otherwise false.
+            friend constexpr bool operator==(const complex<value_type> &z, const complex<value_type> &w);
+            /// Compares complex number z and real y and returns true if they are the same, otherwise false.
+            friend constexpr bool operator==(const complex<value_type> &z, value_type y);
+            /// Compares real x and complex number w and returns true if they are the same, otherwise false.
+            friend constexpr bool operator==(value_type x, const complex<value_type> &w);
+
+            /// Compares complex numbers z and w and returns true if they are different, otherwise false.
+            friend constexpr bool operator!=(const complex<value_type> &z, const complex<value_type> &w);
+            ///Compares complex number z and real y and returns true if they are different, otherwise false.
+            friend constexpr bool operator!=(const complex<value_type> &z, value_type y);
+            /// Compares real x and complex number w and returns true if they are different, otherwise false.
+            friend constexpr bool operator!=(value_type x, const complex<value_type> &w);
+
+            /// Reads a complex number from is.
+            template <class C, class T> friend std::basic_istream<C, T> &operator>>(std::basic_istream<C, T> &is, complex<value_type> &);
+            /// Writes to os the complex number z in the form (real,imaginary).
+            template <class C, class T> friend std::basic_ostream<C, T> &operator<<(std::basic_ostream<C, T> &os, const complex<value_type> &);
+            /// Streams the complex number z in the format "(real,imaginary)" into `sycl::stream` x and return the result.
+            friend const sycl::stream &operator<<(const sycl::stream &x, const complex<value_type> &z);
+
+} // namespace experimental
+} // namespace oneapi
+} // namespace ext
+} // namespace sycl
+```
+
+=== Mathematical operations
+
+This proposal adds to the `sycl::ext::oneapi::experimental` namespace, math
+functions accepting the complex types `complex<sycl::half>`, `complex<float>`,
+`complex<double>` as well as the scalar types `sycl::half`, `float` and `double`
+for the SYCL math functions, `abs`, `acos`, `asin`, `atan`, `acosh`, `asinh`,
+`atanh`, `arg`, `conj`, `cos`, `cosh`, `exp`, `log`, `log10`, `norm`, `polar`,
+`pow`, `proj`, `sin`, `sinh`, `sqrt`, `tan`, and `tanh`.
+
```

Comment:
Add a sentence here too clarifying where these functions are available.  For example:

> These functions are available in both host and device code.